Last interview r-universe blog post (#619)
ropensci/roweb3: content/blog/2023-09-15-r-universe-stars-5/index.md

---
slug: "r-universe-stars-5-en"

> title: "Meeting the Stars of the R-Universe. The R-Universe against diseases."

Reviewer: ```suggestion
title: "Meeting the Stars of the R-Universe: The R-Universe Against Diseases."
```

author:
  - Yanina Bellini Saibene
  - Alejandra Bellini
  - Lucio Casalla  
  - Steffi LaZerte
date: '2023-09-15'
tags:
  - community
  - r-universe
  - interviews
  - r-universe-stars
featured: true
# The summary below will be used by e.g. Mastodon cards
description: "."

---

A new post of our interview series [“Meeting the stars of the R-universe”](/tags/r-universe-stars/). We aim to introduce the teams and people behind the development of software and packages many of us use and which are available through the R-Universe. We want to highlight and explore different teams and projects around the world, the work they do, their processes and users. This stop is at the __United Kingdom__ to talk with members of the Centre for Global Infectious Disease Analysis. Be sure to watch the video at the end with excerpts from the interview.

> _Infectious diseases continue to generate multiple problems in human health. The pandemic experience taught us the importance of research to fight epidemics. Today we will meet a team of developers who use R-Universe to host packages that help in that work._

Reviewer: ```suggestion
_Infectious diseases continue to generate multiple problems in human health. The pandemic experience taught us the importance of research to fight epidemics. Today we will meet a team of developers who use the R-Universe to host packages that help in this work._
```

> The talk begins with Rich FitzJohn, who leads the research software engineering team at the [MRC Centre for Global Infectious Disease Analysis](https://www.imperial.ac.uk/mrc-global-infectious-disease-analysis). This center works on the global analysis of infectious diseases. It is a center for research and development of tools for epidemiological analysis and modeling of infectious diseases to collaborate with the work of national and international agencies to plan public policy responses to potential infectious disease threats.

Reviewer: ```suggestion
The chat begins with [Rich FitzJohn](/author/rich-fitzjohn/), who leads the research software engineering team at the [MRC Centre for Global Infectious Disease Analysis](https://www.imperial.ac.uk/mrc-global-infectious-disease-analysis). This center works on the global analysis of infectious diseases. It is a center for research and development of tools for epidemiological analysis and modeling of infectious diseases to collaborate with the work of national and international agencies to plan public policy responses to potential infectious disease threats.
```

> This team was one of the first to use our R-Universe and their testimony is very relevant to learn and share why they joined our platform.

Reviewer: ```suggestion
This team was one of the first to use our R-Universe and we're interested in understanding and sharing their reasons for joining the platform.
```

> As Rich explains, their work consists of providing answers to general and specific epidemiology problems and responding to more than 200 different companies. Within these developments, R-Universe is a way to distribute their packages mainly for internal work and a good way to present them.

Reviewer: The following paragraphs suggest that the packages (or at least some) aren't strictly for internal use, so is how I've modified the sentence okay?
```suggestion
As Rich explains, their work consists of providing answers to general and specific epidemiology problems and responding to more than 200 different companies. Within this framework, they use the R-Universe as a way to distribute and present packages they develop and use in their work.
```

Author: Yes, it is correct.

> In this sense, Rob, a software engineer who works with John, adds that the packages allow them to deliver the work to all the users in a simple, useful, and responsible way. Especially if you consider that it is very collaborative work and many times against the clock. That is why they try to be all working at the same time on the same code with a permanent development and revision cycle to meet the needs of the researchers.

Reviewer: Introduce Rob fully? And it should be Rich, right? Not John...?
```suggestion
Rob LASTNAME, a software engineer who works with Rich, adds that these packages are important as they allow them to deliver the work to all the users in a simple, useful, and responsible way. Especially if you consider that it is very collaborative work and often they are working against the clock. That is why they try to all work at the same time on the same code with a permanent development and revision cycle to meet the needs of the researchers.
```

> This development can then be used by researchers worldwide, and that is where R-Universe becomes a fundamental tool.

Reviewer: ```suggestion
This work will then be used by researchers worldwide, and that is where the R-Universe becomes a fundamental tool.
```

> Multiple users can access the [MRC universe](https://mrc-ide.r-universe.dev) to use the packages and have the necessary material available to develop some type of research. However, it is likely that they will need some training or knowledge to be able to use them because of the level of specificity. R-Universe is also a place to see what scientists are studying and developing because they also use the platform to upload their packages and find other packages to use in their research. 

Reviewer: I felt like the "training" section needed a bit of a conclusion (is it correct?). I also felt like that final sentence was a bit out of place. Not incorrect but it didn't seem to fit in with the topic being discussed.
```suggestion
Multiple users can access the [MRC R-Universe](https://mrc-ide.r-universe.dev) at the same time to use the packages and have the necessary material available for their research. However, it is likely that they will need some training or knowledge to be able to use these packages because of the level of specificity, and the R-Universe helps there too by hosting documentation such as vignettes and articles.
```

Author: I understand.  They are really like two different things.  The first part has to do with their universe on the platform.  The other is how sometimes they use r-universe to look for other packages and see what other people are doing.

> Recalling how they first entered the R-universe, Rich says it was by trying out a project that rOpenSci was working on.  When they were invited, they realized that it represented the possibility of saving hours of work in different processes. Over time the features and functionalities of R-Universe changed and expanded.

Reviewer: ```suggestion
Recalling how they first entered the R-universe, Rich says it was by being invited to try out a new project that rOpenSci was working on. They then quickly realized that it represented the possibility of saving hours of work in different tasks. Over time the features and functionalities of the R-Universe changed and expanded as the project matured.
```

> It was at this point in the talk that they discussed possible future features that could be useful in R-Universe; from archiving different versions of the binaries of each package according to a researcher's reproducibility needs, to automatic generation of package documentation, to different ideas for increasing R-Universe's package search and discovery capabilities.

Reviewer: ```suggestion
It was at this point in the interview that Rich and Rob discussed possible future features that could be useful in the R-Universe; from archiving different versions of the binaries of each package according to a researcher's reproducibility needs, to automatic generation of package documentation, to different ideas for increasing the R-Universe's package search and discovery capabilities.
```

> Some of these challenges are under development and R-Universe already has features that respond to these ideas from Rich and Rob's team. For example: [Downloading snapshots and creating stable R packages repositories using r-universe](https://ropensci.org/blog/2023/05/31/runiverse-snapshots/) and [Discovering and learning everything there is to know about R packages using r-universe](https://ropensci.org/blog/2023/02/27/runiverse-discovering/).

Reviewer: ```suggestion
Some of these features are currently under development and others have already been implemented in response to these ideas from Rich and Rob's team. For example: [Downloading snapshots and creating stable R packages repositories using R-Universe](/blog/2023/05/31/runiverse-snapshots/) and [Discovering and learning everything there is to know about R packages using R-Universe](/blog/2023/02/27/runiverse-discovering/).
```

> In the following video, Rich and Rob share more details of their work in the MRC and share in the rOpenSci R-Universe.

Reviewer: ```suggestion
In the following video, Rich and Rob share more details of their work in the MRC and with the rOpenSci R-Universe.
```

> {{< vimeo id=829850626 title="Meeting R-Universe Stars - xxxxx project" >}}

Reviewer: Wrong video?

Author: Yes!
